dbt/adapters/snowflake/relation_configs/dynamic_table.py

```diff
@@ -21,6 +35,147 @@ class SnowflakeDynamicTableConfig(RelationConfigBase):
     """
 
     name: str
+    schema_name: str
```

Comment:
True, I did that in cases when there are defaults because passing in `my_arg=None` does not result in the default for `my_arg`, it results in `None`. However, I don't have defaults. I'll remove those guards below.